contracts/strategies/aura/AuraStrategystETH.sol

```diff
@@ -0,0 +1,52 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity 0.5.16;
+pragma experimental ABIEncoderV2;
+
+import "./base/AuraStrategyUL.sol";
+
+contract AuraStrategystETH is AuraStrategyUL {
```

Comment:
For the implemented strategy contracts, we prefer to fill out all the values within this contract instead of inputting them with the initialisation. As you can see in other strategy contracts in this repository, the only fields generally open for input in the `initializeStrategy` function are `_storage` and `_vault`. All other values can be filled within this contract.

The reason is that there is additional room for error when we need to pass these values with the function call, and there is no reason for these values to change between development and testing and actual deployment.

With filling in these values also comes our naming convention, which would in this case be `AuraStrategyMainnet_stETH`. Mainnet denoting that the implementation contains the values for Mainnet deployment. We don't really develop for any testnets, but we kept the convention this way anyways.

So please include all the values within this contract and let me know if you have any questions about this :)